src/components/Header/HeaderNavMobile.svelte

```diff
@@ -10,11 +10,11 @@
   class={`z-1 fixed right-0 top-0 z-10 w-[100vw] flex-col justify-between gap-6 overflow-y-scroll bg-navy px-[12%] font-title transition-[height] duration-[1s] sm:flex md:hidden lg:hidden ${isOpen ? 'h-[100vh]' : 'h-0'}`}
   aria-expanded={isOpen}
 >
-  <nav id="mobile-navigation" aria-label="Main" class="mt-[60px] flex flex-col gap-4 text-2xl uppercase text-cream">
+  <nav id="mobile-navigation" aria-label="Main" class="text-cream mt-[60px] flex flex-col gap-4 text-2xl uppercase">
     <NavItems textColor="cream" bind:section isDesktop={false} />
   </nav>
 
-  <div class="z-50 pb-[40px] font-sans text-cream">
+  <div class="text-cream z-50 pb-[40px] font-sans">
```

Comment:
double checking that cream is still a value being used. I know it was taken out of the tailwind config, so currently I don't think this is doing anything.